pkg/config/application.go

```diff
@@ -66,6 +67,48 @@ func NewApplicationGroup(application *Application, mspConfig *mb.MSPConfig) (*cb
 	return applicationGroup, nil
 }
 
+// RemoveAnchorPeer removes an anchor peer from an existing channel config transaction.
+// The removed anchor peer and org it belongs to must both already exist.
+func RemoveAnchorPeer(config *cb.Config, orgName string, anchorPeerToRemove *AnchorPeer) error {
+	if _, orgExists := config.ChannelGroup.Groups["Application"].Groups[orgName]; !orgExists {
```

Comment:
You might want to consider using `channelconfig.ApplicationGroupKey` instead of inline "Application" const.

Reply:
good catch, forgot about that